### DISCUSSION TITLE
Quality score understanding

### DISCUSSION THREAD

**Discussion by seemantsingh-code** [guillermo-navas-palencia/optbinning#289] [Q&A] [answered]
Can anyone give a brief explanation or any resource to understand what all aspects related to the variables are captured in the final quality score

**Comment by guillermo-navas-palencia** [comment, marked as answer]
Hi @maxsanwal. 

The binning quality score is described in https://arxiv.org/pdf/2001.08025.pdf section 3.3 (pages 17 and 18).
